src/cloud/validateCode.js

```diff
@@ -164,17 +169,25 @@ const validateCode = async request => {
       }
 
       if (reservationId) {
-        const reservation = await ReservationService.claimReservation(reservationId, user);
+        const reservation = await ReservationService.claimReservation(
+          reservationId,
+          user,
+        );
         const conversationCid = reservation.get('conversationId');
         const conversation = await ChatService.getConversationByCid(
           conversationCid,
         );
 
-        const fromUser = await new Parse.Query(Parse.User).get(conversation.data.created_by.id);
+        const fromUser = await new Parse.Query(Parse.User).get(
+          conversation.data.created_by.id,
+        );
         const fullName = UserUtils.getFullName(fromUser);
-        const connection = await new Parse.Query('Connection').equalTo('channelSId', conversationCid).find({ useMasterKey: true });
+        const connection = await new Parse.Query('Connection')
+          .equalTo('channelSId', conversationCid)
```

Comment:
ChannelCid not ChannelSid

Reply:
sorry this is initialConversations, which is an array of strings

Reply:
conversationSid is no longer used.

Reply:
This code is for reservationId not for waitlist